techniques/flash/FLASH3.html

```diff
@@ -22,9 +22,9 @@ <h3>Hiding a graphic in the Flash Professional authoring tool</h3>
       <section class="example">
          <h3>Applying textual alternatives programmatically in ActionScript 2.0</h3>
          
-            <p>To manage an object's text equivalents programmatically using  ActionScript, the <code class="prop">_accProps</code> property must be used. This references an  object containing accessibility related properties set for the object.  The code example below shows a simple example of how the <code class="prop">_accProps</code>  property is used to remove an object from the accessibility information  for the movie using ActionScript. </p>
+            <p>To manage an object's text equivalents programmatically using  ActionScript, the <code>_accProps</code> property must be used. This references an  object containing accessibility related properties set for the object.  The code example below shows a simple example of how the <code>_accProps</code>  property is used to remove an object from the accessibility information  for the movie using ActionScript. </p>
          
-         <pre xml:space="preserve">// 'decorative_mc' is an instance placed on the movie's main timeline
+         <pre>// 'decorative_mc' is an instance placed on the movie's main timeline
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`